app/templates/search/_summary_accessible_hint.html

```diff
@@ -0,0 +1,3 @@
+<div id="js-dm-live-search-summary-accessible-hint">
+    {{ total }} {{ pluralize(total, "result", "results") }} found
```

Comment:
This is where I wanted the two-space indent(!)

Reply:
huh yeah not sure why my editor didn't. sometimes it doesn't quite detect the syntax properly.